modules/web/sites/slots/hybrid-connection-namespaces/relays/main.bicep

```diff
@@ -0,0 +1,61 @@
+@description('Required. The resource ID of the relay namespace hybrid connection.')
+param hybridConnectionResourceId string
+
+@description('Required. Slot name to be configured.')
+param slotName string
+
+@description('Conditional. The name of the parent web site. Required if the template is used in a standalone deployment.')
+param appName string
+
+@description('Optional. Name of the authorization rule send key to use.')
+param sendKeyName string = 'defaultSender'
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+var splitResourceId = split(hybridConnectionResourceId, '/')
```

Comment:
Please comments above regarding this variable

Reply:
Standardized